### DISCUSSION TITLE
Map style for hiking pathes

### DISCUSSION THREAD

**Discussion by MarcTasman** [osmandapp/OsmAnd#11569] [Maps Related]
Dear OsmAnd community,

I am trying to create my own map style. However, how do I address the different line (point...) features? I do not find a translation between the www.openstreetmap.org/edit feature names (like highway: path, surface: ground, foot: yes ...) and the “default.render.xml” file. Unfortunately, the documentation in the “default.render.xml” file did not help me to answer my questions completely. Is there some other documentation somewhere?

My idea (I attached my map style file) was to use for all walking paths orange and for all major roads grey.
Attached you find an example of the style I already created, but how can I change the line sections 1a / 1b / 3 / 4 / 5 (see page 1) - the www.openstreetmap.org/edit feature attributes of each section you can see on the screenshots (page 2 - 7) and on the last screenshots the “normal / default.render.xml” map style (page 8).

In addition, a) how do I adjust the thickness of the lines (in the different zoom modes), b) switch off the naming of the line (road) features from zoom level 12 and c) prevent the app to switch to the standard map style in navigation mode?

Thanks a lot for your answer and the time you spent in the great OSMand project for all of us.
Best wishes
Marc

[2021-04-29_Example map style.pdf](https://github.com/osmandapp/OsmAnd/files/6408407/2021-04-29_Example.map.style.pdf)
[2021-04-29_LightRS_v02.render.xml.zip](https://github.com/osmandapp/OsmAnd/files/6408409/2021-04-29_LightRS_v02.render.xml.zip)

**Comment by vshcherb** [comment]
Recently I was writing a bigger article how rendering style rules are evaluated - https://docs.osmand.net/en/main@latest/development/osmand-file-formats/osmand-rendering-style#map-style-sections. Please double check if there are no answers to your questions there. If you could post questions somehow in a format, so it could be contributed back to documentation it could be beneficial for your case and future questions.

**Comment by MarcTasman** [comment]
Dear vshcherb,

thanks for your answer - I will have a look on the page you mentioned.

Best wishes
Marc
